pytest_odoo.py

```diff
@@ -79,6 +78,13 @@ def pytest_cmdline_main(config):
         yield
 
 
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
Any reason to make it module level than session level?

Reply:
Yes, it seems that server is killed when entering testing functions using session.